src/plugins/intel_gpu/src/plugin/graph.cpp

```diff
@@ -367,6 +367,10 @@ std::shared_ptr<ngraph::Function> Graph::GetExecGraphInfoByPrimitivesInfo(std::v
             auto param = std::make_shared<ngraph::op::Parameter>(out_et, out_pshape);
             params.push_back(param);
             return_node = param;
+            // create additional result node if parameter is output without post reorder
+            if (is_output) {
```

Comment:
Where did you observed such case parameter == is_output?

Reply:
![image](https://github.com/openvinotoolkit/openvino/assets/47554900/0829c01b-125f-4b7b-b640-d67b16d3e116)
In some test case of FakeQuantizeTransformation, there was case where parameter was also output directly by not inserting the post reorder.